### PR TITLE
Update autocomplete list to add IDMS statements

### PR DESCRIPTION
**Pull request by zeibura**
Updated the list of things supported by autocomplete to include CA IDMS DML statements; also compacted the list somewhat as it was getting a bit long.
For the next release.

Signed-off-by: Zeibura Kathau <zeibura.kathau@broadcom.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +40,9 @@ COBOL Language Support provides the following COBOL syntax awareness features:
 ### Autocomplete
 Autocomplete speeds up the coding process by intuitively suggesting the most likely variables or paragraphs to follow existing code. The extension provides live suggestions while you type for:
 
-- COBOL keywords
-- COBOL variables
-- COBOL paragraphs
-- CICS keywords
-- DB2 SQL keywords
+- COBOL keywords, variables and paragraphs
+- CA IDMS DML statements
```
Comment (temanbrcom) [inline README.md:44]: Technically, it is also keywords.

File: README.md [hunk 2]
```diff
@@ -40,11 +40,9 @@ COBOL Language Support provides the following COBOL syntax awareness features:
...
+- CICS and DB2 SQL keywords
```
Comment (temanbrcom) [inline README.md:45]: I think here we may mention the variables too.

Comment (asatklichov) [reply README.md:45]: I guess we need to add 'statements" to both CICS and SQL. It supports writing those statements at the end. 
e.g.  
- CA IDMS DML keywords, statements
- CICS keywords, statements
- DB2 SQL keywords, DML and DDL statements

Comment (temanbrcom) [reply README.md:45]: It is a list of autocomplete elements only. Do we provide also the snippets for SQL and CICS? IDMS definitely no, it is still in discussion.

Comment (asatklichov) [reply README.md:45]: Yes for both we provide snippets, dare to mention for sure

**Comment by temanbrcom** [comment]
I suggest this:

```
- COBOL statements, keywords, variables, paragraphs, and sections
- CICS and DB2 SQL keywords and statements
- CA IDMS DML keywords
- ```

**Comment by zeibura** [comment]
Re the COBOL sections, aren't those code snippets? bear in mind code snippets are already mentioned

**Comment by temanbrcom** [comment]
> Re the COBOL sections, aren't those code snippets? bear in mind code snippets are already mentioned

Sections are hierarchical elements of the division, so they should be mentioned. Thanks, I didn't notice that the snippets are already mentioned. So, the changes may look like this:
```
- COBOL keywords, variables, paragraphs, and sections
- CICS and DB2 SQL keywords
- CA IDMS DML keywords
```

**Comment by zeibura** [comment]
OK, I've added "sections" under COBOL.

**Review by temanbrcom — APPROVED** [review]
🐝

**Review by asatklichov — APPROVED** [review]
🎲

**Comment by temanbrcom** [comment]
I also noticed that there is `sections` missing here:
```
Syntax Coloring
Contrasting colors are used in displayed code for ease of identifying and distinguishing keywords, variables, and paragraphs.
```

**Comment by zeibura** [comment]
added "sections" under syntax coloring

**Review by temanbrcom — APPROVED** [review]

**Review by zimlu02 — APPROVED** [review]

**Pull request merged**